tests/Google/Ads/GoogleAds/Lib/ConfigurationLoaderTestProvider.php

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-namespace Google\Ads\GoogleAds\Lib\Testing;
+namespace Google\Ads\GoogleAds\Lib;
```

Comment:
I think we should still somehow separate these helping files from the rest of normal unit tests.
And then add a rule in phpcs_ruleset.xml to exclude them.
The folder name may not need to be `Testing`, but it should indicate it's the place for it's test helpers.

Currently, if a user runs phpunit without specifying the directories, these files will be unnecessarily included.
Although it would work, it includes too many files than it needs.

Reply:
Why should we exclude them from code style? I believe that those are not excluded today and are compliant.

I think this would add complexity for small-to-no benefits. Did you keep a link to a well designed implementation that does this? If  we go with this solution, I think of two options: add a new directory next to `tests` or have a `Google\Ads\GoogleAds\Testing` package so that all those classes are at the same location, any preference on your side?

Reply:
Sorry. I don't mean the code style checking, but the PHPUnit testing.
With your structure, can it separate what are really unit tests and what are not?

Reply:
NVM. I think we can use something similar to [this line](https://github.com/sebastianbergmann/comparator/blob/master/phpunit.xml#L15) to filter those files.
Let me TAL at your PR one more time.

Reply:
All look fine. Could you modify phpunit rules to be more specific?

Reply:
I refreshed the phpunit configuration file: I removed unnecessary coverage whitelist exclusions and I added the `Test.php` suffix filter as you suggested.